t/planlib/TestsFor/Attributes/Subclass.pm

```diff
@@ -112,12 +112,14 @@ sub expected_test_events {
                     event Ok => sub {
                         call pass => T();
                         call name =>
-                          q{Overriding and calling parent, but we don't have a plan and parent does: 1},
+                          q{Overriding and calling parent, but we don't have a plan and parent does: 1}
```

Comment:
I've tidied all the code in master so this sort of thing shouldn't be necessary.